src/ray/raylet/node_manager.cc

```diff
@@ -1010,16 +1102,21 @@ ray::Status NodeManager::ForwardTask(const Task &task, const ClientID &node_id)
           ObjectID argument_id = spec.ArgId(i, j);
           // If the argument is local, then push it to the receiving node.
           if (task_dependency_manager_.CheckObjectLocal(argument_id)) {
-            RAY_CHECK_OK(object_manager_.Push(argument_id, node_id));
+            if (!object_manager_.Push(argument_id, node_id).ok()) {
```

Comment:
@elibol currently it looks like `Push` always succeeds. Is there a scenario where `Push` would fail? If not, then I think we should just have `Push` return `void`, which seems fine since it is best effort anyway.

Reply:
I agree; `Push` ought to be of type `void`.

Reply:
Ok cool, I'll make that change.